libs/EXTERNAL/libtatum/libtatum/tatum/TimingConstraintsFwd.hpp

```diff
@@ -2,7 +2,7 @@
 
 namespace tatum {
 
-//Forward delcaration
+//Forward declaration
```

Comment:
Similarly everything under libs/EXTERNAL is also developed externally (and included in VTR usually with git-subtree). 

If you want you can submit these Tatum related changes to www.github.com/kmurray/tatum instead, and they will end up in VTR the next time we update Tatum's subtree